python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -840,6 +840,15 @@ def do_evaluate(
         return DataFrame(columns)
 
 
+class GroupbyOptions:
+    """Serializable wrapper for polars GroupbyOptions."""
+
+    def __init__(self, polars_groupby_options: Any):
+        self.dynamic = polars_groupby_options.dynamic
+        self.rolling = polars_groupby_options.rolling
```

Comment:
Eventually these will also need translated, but for now we can dodge it because they are always `None`.

Reply:
Indeed, I couldn't find any examples in our tests to aid that, so for now I left it as above to make things simpler.